internal/mode/static/state/dataplane/types.go

```diff
@@ -19,6 +19,11 @@ const (
 	PathTypeExact PathType = "exact"
 )
 
+const (
+	// KubeSystem indicates the name of kube-system namespace
+	KubeSystem = "kube-system"
```

Comment:
Since this is only used in the telemetry package, I would define this constant there.

Reply:
It also doesn't need to be exported if it's only used in the telemetry package